src/Components/Web.JS/src/Platform/BootConfig.ts

```diff
@@ -22,6 +22,7 @@ export class BootConfigResult {
 export interface BootJsonData {
   readonly entryAssembly: string;
   readonly resources: ResourceGroups;
+  /** Gets a value that determines if this boot config was produced from a non-published build (i.e. dotnet build or dotnet run) */
```

Comment:
nit: is /** */ the extra star intentional?

Reply:
Yup. It's how you author a JSDoc: https://jsdoc.app/about-getting-started.html